Training/2.0 Libraries.md

```diff
@@ -1,65 +1,60 @@
 # Libraries
 
-First of all if you haven't done so already download Anaconda here: https://www.anaconda.com/products/individual . Anaconda has a bunch of libraries and functionality installed which will really help us with our data analytics project. We will be using jupyter notebooks throughout this training period. 
+First of all if you haven't done so already download Anaconda here: https://www.anaconda.com/products/individual . Anaconda has a bunch of libraries and functionality installed which will really help us with our data science project. We will be using jupyter notebooks throughout this training period. 
 
-To start getting familiar with some useful data analytics libraries, we will need to know how to manipulate data. For this we will be using NumPy and Pandas libraries. 
-
-Documentation for NumPy: https://numpy.org/doc/stable/
-Documentation for Pandas: https://pandas.pydata.org/docs/
+To start us getting familiar with some useful data science libraries we will need to know manipulate data. For this we will be using NumPy and Pandas libraries. 
 
 ### Task 1: 
 
 Import the numpy library and make a key word for it: np 
 Import the pandas library and make a key word for it: pd
 
 ```
-
+import numpy as np
+import pandas as pd
 
 ```
 
 
 When to use which? Each of these libraries has very useful functionality but you need to know when to use them. 
 
-### Task 2a)
+### Task 2
 Make a two-dimensional numpy array. First row will be: [1,2,3] and second row will be [4,5,6]. 
 Using numpy functionality find the mean, minimum and max value of your array. Index the maximum value and assign it to a variable named max. Index the minimum value and assign it to a variable named min. 
 
 ```
-
+import numpy as np
+arr=np.array([[1,2,3],[4,5,6]])
+maxx=np.amax(arr)
+minn=np.amin(arr)
 ```
 
-### Task 2b)
-Calculate the trimmed mean of the following array using ONLY Numpy and python's base functions: [67,67,65,50,69,68,67,70,69,66]. The trimmed mean is the mean of the data with the largest and the smallest values (outliers) removed.
-
-```
-
-```
 
```

Comment:
If you can, have a go at this question. You can use your answers from part a) as guidance. If you're still struggling, let either Hasitha or myself know so we can walk you through the answer